xml/admin_gui_dashboard.xml

```diff
@@ -653,16 +666,17 @@
   <sect2 xml:id="dashboard-editing-roles">
    <title>Editing Custom Roles</title>
    <para>
-    Click a custom role's table row and then select <guimenu>Edit</guimenu> in
+    Click a user's table row to highlight selection. Select <guimenu>Edit</guimenu> in
```

Comment:
Again: surely "highlight _the_ selection" or "highlight _your_ selection", no? 

Is it just me but "click to highlight selection" sounds strange. The phrase "highlight selection" is not a verb (that I know of): it seems to be saying "click to X the Y," or "click to X your Y", no?